content/kubeone/master/guides/all_in_one_node/_index.en.md

```diff
@@ -0,0 +1,63 @@
++++
+title = "All In One Node"
+date = 2022-03-21T12:13:28+02:00
+enableToc = true
++++
+
+Sometimes it's needed to create a one-node-cluster, which will host control
+plane components and as well the usual workloads. It's possible to do so with
+small tweaks.
+
+The setup of "one node fits all" or all-in-one in general is a usual KubeOne
+cluster, the only difference would be: removed [taints][taints-glossary] from the Node.
+
+## What
```

Comment:
Nit: let's use some more descriptive headers. It makes it easier for readers but also helps with SEO.

Example: Introduction, default setup...